src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs

```diff
@@ -0,0 +1,2105 @@
+        public static event EventHandler<DialogRequestedEventArgs> DialogRequested
+        {
+            add
+            {
+                _dialogRequestDelegate = (int pid, IntPtr dispText, IntPtr uttText, bool continuous, IntPtr userData) =>
+                {
+                    string dispTextString = Marshal.PtrToStringAnsi(dispText);
+                    string uttTextString = Marshal.PtrToStringAnsi(uttText);
+                    DialogRequestedEventArgs args = new DialogRequestedEventArgs(pid, dispTextString, uttTextString, continuous);
+                    _dialogRequested?.Invoke(null, args);
+                };
+                ErrorCode error = VcMgrSetDialogRequestCb(_dialogRequestDelegate, IntPtr.Zero);
+                if (error != ErrorCode.None)
+                {
+                    Log.Error(LogTag, "Add DialogRequested Failed with error " + error);
+                }
+                else
+                {
+                    _dialogRequested += value;
+                }
+            }
+
+            remove
+            {
+                ErrorCode error = VcMgrUnsetDialogRequestCb();
+                if (error != ErrorCode.None)
+                {
+                    Log.Error(LogTag, "Remove DialogRequested Failed with error " + error);
+                }
+
+                _dialogRequested -= value;
+                _dialogRequestDelegate = null;
+            }
+        }
+
+        /// <summary>
+        /// Enable command type as candidate command.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>
+        /// http://tizen.org/privilege/voicecontrol.manager
+        /// </privilege>
+        /// <privlevel>
+        /// public
+        /// </privlevel>
+        /// <feature>
+        /// http://tizen.org/feature/speech.control
+        /// http://tizen.org/feature/microphone
+        /// </feature>
+        /// <param name="cmdType">Command Type</param>
+        /// <exception cref="InvalidOperationException">This exception can be due to invalid state.</exception>
+        /// <pre>
+        /// The State should be Ready
+        /// </pre>
+        public static void EnableCommandType(CommandType cmdType)
+        {
+            ErrorCode error = VcMgrEnableCommandType(cmdType);
+            if (error != ErrorCode.None)
+            {
+                Log.Error(LogTag, "EnableCommandType Failed with error " + error);
+                throw ExceptionFactory.CreateException(error);
+            }
+        }
+
+        /// <summary>
+        /// Disable command type as candidate command.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>
+        /// http://tizen.org/privilege/voicecontrol.manager
+        /// </privilege>
+        /// <privlevel>
+        /// public
+        /// </privlevel>
+        /// <feature>
+        /// http://tizen.org/feature/speech.control
+        /// http://tizen.org/feature/microph
```

Comment:
If this exception can be thrown in both cases, please write it as a single instance of `<exception cref="InvalidOperationException">`.